crates/sallyport/src/guest/call/enarxcall/passthrough.rs

```diff
@@ -101,36 +101,50 @@ impl PassthroughAlloc for MemInfo {
     }
 }
 
-/// Within an address range inside the enclave, ask host to remove pages
-/// from the enclave. Pages must be trimmed before this operation is
-/// applied.
-pub struct RemoveSgxPages {
+/// Notify the host about `mmap()`.
```

Comment:
Add context here, is this notifying the host about a `mmap` issued and handled by the guest eariler?
If so, use a reference to `Mmap` struct here, i.e. `[Mmap](crate::guest::call::syscall::Mmap)`

Reply:
fixed